systemd/user/kano-common-refresh-token.timer

```diff
@@ -0,0 +1,16 @@
+#
+# kano-common-refresh-token.timer
```

Comment:
If I remember correctly, this timer needs to be activated after installation in order for it to start running.

Reply:
My tests reveal that this is not necessary, because it has been added to the `kano-common.target` service, and it starts triggering automatically. There is a short explanation on the headers for this.